doc/reference/ovn-internals.md

```diff
@@ -0,0 +1,168 @@
+
+For managed `physical` networks it is the per-cluster member value of the `parent` setting.
+The `parent` interface itself can have one of three types:
+
+- Linux native bridge.
+- OVS Bridge.
+- Physical interface (or bond).
+
+It is important to note that a `physical` managed network's `parent` interface can be any of these types, and that for a `managed` bridge network the parent interface can be either types of bridges.
+
+### Bridge (OpenVSwitch)
+A user can separately configure a managed bridge network with the `openvswitch` `bridge.driver`.
+An OVN network can be created with `network` set to the name of the managed bridge network.
+In this case LXD configures a bridge mapping on the OVS bridge to connect the OVN network:
+
+```{figure} /images/ovn/ovn-uplink-bridge-ovs.svg
+OVN uplink OVS bridge
+```
+
+### Physical
+An OVN network can be created with `network` set to a physical network, where the physical network is essentially a database entry in LXD that tells it how to interact with an actual `parent` network device.
+In this case, an OVS bridge is created automatically.
+A bridge port connects the OVS bridge to the parent.
+A bridge mapping is used (as above) to connect the OVS bridge to the OVN network.
+
+```{figure} /images/ovn/ovn-uplink-physical.svg
+OVN uplink physical
+```
+
+```{note}
+When using a physical network as an uplink for OVN, any IP addresses on the parent interface will become defunct.
+The parent network must not have any assigned IP addresses.
+```
+
+### Bridge (native)
+A native Linux bridge can be used.
+In this case, we perform the same steps as in the physical network and additionally configure a `veth` pair.
+The `veth` pair is used so that the bridge can still be used for other purposes (since the bridge maintains its configuration).
+This is handy for development and testing but is not performant and should not be used in production.
+
+```{figure} /images/ovn/ovn-uplink-bridge-native.svg
+OVN uplink native bridge
+```
+
+## OVN Network
+In the simplest case, a LXD OVN network has the below configuration:
+
+```{figure} /images/ovn/ovn-network.svg
+:width: 100%
+
+LXD OVN network diagram
+```
+
+```{note}
+This diagram does not show cross-cluster networks.
+This conceptual diagram should look the same on all cluster members.
+If the chassis group prioritizes another chassis for the uplink, the traffic is routed that chassis.
+```
+
+## Integration bridge
+The cluster setting `network.ovn.integration_bridge` must contain the name of an OVS bridge that is used to connect instances to an OVN network via a NIC device.
+This OVS bridge must be pre-configured on all cluster members with the same name.
+Connectivity to the integration bridge differs between containers and virtual machines:
+
+- Containers use a `veth` pair (similar to connecting to a native bridge uplink network).
+
+    ```{figure} /images/ovn/ovn-integration-bridge-container.svg
+    Integration bridge connectivity with containers
```

Comment:
In that diagram, between the `Veth pair`, is the VLAN filter also applied lik in the native bridge uplink network case?

Reply:
I couldn't see any evidence of that in the code. What are your thoughts @tomponline?